go/vt/orchestrator/logic/topology_recovery.go

```diff
@@ -1370,16 +1370,10 @@ func executeCheckAndRecoverFunction(analysisEntry inst.ReplicationAnalysis, cand
 	// changes, we should be checking that this failure is indeed needed to be fixed. We do this after locking the shard to be sure
 	// that the data that we use now is up-to-date.
 	if isActionableRecovery {
-		ksInfo, err := ts.GetKeyspace(ctx, analysisEntry.AnalyzedKeyspace)
+		err := RefreshKeyspace(analysisEntry.AnalyzedKeyspace)
 		if err != nil {
 			return false, nil, err
 		}
-		if ksInfo.GetDurabilityPolicy() != "" && ksInfo.GetDurabilityPolicy() != config.Config.Durability {
-			err = log.Errorf("executeCheckAndRecoverFunction: Analysis: %+v, InstanceKey: %+v: Keyspace has a different durability policy configured than this vtorc instance: %v",
-				analysisEntry.Analysis, analysisEntry.AnalyzedInstanceKey, analysisEntry.AnalyzedKeyspace)
-			return false, nil, err
-		}
-
```

Comment:
We no longer need this check, since we are now refreshing the keyspace which will reload the durability policy if it has changed, followed by a check for if failure is actually required.